1.9/usage/managing-services/task-handling/index.md

```diff
@@ -0,0 +1,83 @@
+---
+post_title: Task Handling
+nav_title: Task Handling
+menu_order: 1.5
+---
+
+Marathon sorts tasks into three categories of states: initial, non-terminal, and terminal. Tasks within these categories may have one of several statuses, as summarized in the diagram below.
```

Comment:
The number `three` here is not important. how about `Marathon sorts tasks into these categories:`